src/index.ts

```diff
@@ -25,15 +25,36 @@ type Preferences = {
   squashMergeAllowed?: boolean,
 }
 
-const getToken = async () => {
+// Prompt the user for a token and return it as a string
+const tokenPrompt = async () => {
   console.log('Create a token at https://github.com/settings/tokens/new?scopes=repo')
-  return (await prompts({
+  const response = await prompts({
     type: 'password',
     name: 'token',
     message: 'GitHub access token:',
-    validate: (value: string) => value.startsWith('ghp_') ? true : 'Your token should start with ghp_',
-  })).token;
-}
+    validate: (value: String) =>
```

Comment:
```suggestion
    validate: (value: string) =>
```

This should remain `string`
https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string